src/g3log/time.hpp

```diff
@@ -55,6 +56,12 @@ namespace g3 {
    * std::put_time. A possible fix if your c++11 library is not updated is to
    * modify this to use std::strftime instead */
    std::string localtime_formatted(const system_time_point& ts, const std::string& time_format) ;
+
+   inline system_time_point to_system_time(const high_resolution_time_point& ts)
+   {
+	   using namespace std::chrono;
+	   return time_point_cast<system_clock::duration>(system_clock::now() + (ts - high_resolution_clock::now()));
```

Comment:
The reason why it was changed earlier was that on VMs this creates funky time issues as reported by @Jawan81 in https://github.com/KjellKod/g3log/issues/175

I have not been able to reproduce that issue with this pull request. 
@Jawan81 can you try this pull request in the environments you saw the time issues in to verify that this does not break it?